### DISCUSSION TITLE
Debit Cards are being declined for no reason !?

### DISCUSSION THREAD

**Discussion by Himanshu-prime** [planetscale/discussion#532] [FAQs] [answered]
The VISA/Mastercard debit cards are being declined there isn't any bank side issue the same cards are working absolutely fine everywhere but arent being accepted on planetscale !

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @Himanshu-prime, 

Could you send an email in to support: https://support.planetscale.com/hc/en-us? We aren't able to debug card authorization issues in public.
